packages/thirdweb/src/extensions/erc7702/account/createSessionKey.ts

```diff
@@ -0,0 +1,163 @@
+import type { BaseTransactionOptions } from "../../../transaction/types.js";
+import { randomBytesHex } from "../../../utils/random.js";
+import type { Account } from "../../../wallets/interfaces/wallet.js";
+import {
+  createSessionWithSig,
+  isCreateSessionWithSigSupported,
+} from "../__generated__/MinimalAccount/write/createSessionWithSig.js";
+import {
+  type CallSpecInput,
+  CallSpecRequest,
+  ConstraintRequest,
+  SessionSpecRequest,
+  type TransferSpecInput,
+  TransferSpecRequest,
+  UsageLimitRequest,
+} from "./types.js";
+
+/**
+ * @extension ERC7702
+ */
+export type CreateSessionKeyOptions = {
+  /**
+   * The admin account that will perform the operation.
+   */
+  account: Account;
+  /**
+   * The address to add as a session key.
+   */
+  sessionKeyAddress: `0x${string}`;
```

Comment:
for inputs generally we do string. internally you can use `getAddress(x)` to convert it to a 0xstring type (and validate its an address)